cms.php

```diff
@@ -1,72 +1,69 @@
 <?php
+$db = new PDO('mysql:host=127.0.0.1; dbname=jackdb', 'root');
 
-?>
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$query= $db->prepare("SELECT `email`, `phone`, `about`,`deleted`
+FROM `cms` WHERE `deleted` = 0 ORDER BY `id` DESC LIMIT 1;");
+
+$query1= $db->prepare("SELECT `project_id`, `project_img`, 
+`project_title`,`project_text`, `project_link` FROM `projects` WHERE 
+`project_delete` = 0 ORDER BY `project_id` ASC;");
+
+$query->execute();
+$query1->execute();
 
+$result = $query->fetchAll();
+$result1 = $query1->fetchAll();
+
+$email = $result [0]['email'];
+$phone = $result [0]['phone'];
+$about = $result [0]['about'];
+$del = $result [0]['deleted'];
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="cms.css">
+    <title>Portfolio Input Page</title>
+</head>
 <h3>Welcome to the Portfolio Input Page</h3>
 
-<form method="post" action="#">
-    <input type="text" name="email">email
+<form method="post" action="cms.php">
+    <input type="text" value="<?php echo $email; ?>" name="email">email
     <input type="submit" value="update">
 </form><br>
-<form method="post" action="#">
-    <input type="tel" name="tel">Telephone Number
+<form method="post" action="cms.php">
+    <input type="tel" value="<?php echo $phone; ?>" name="tel">Telephone Number
     <input type="submit" value="update">
 </form><br>
-<form method="post" action="#">
-    <input type="text" name="about">About me paragraph
+<form method="post" action="cms.php">
+    <textarea rows="8" cols="150"  name="about" form="about">
+        <?php echo $about; ?></textarea>
```

Comment:
Suggest to move</textarea> to next line fro clarity.

Reply:
done